main.go

```diff
@@ -31,7 +31,7 @@ func main() {
 	pflag.StringVar(&githubappConfig.App.PrivateKey, "github-private-key", "", "github app private key content")
 
 	messageTemplate := pflag.String("message-template", "'{{.Branch}}' will auto-release to: {{range .AutoReleaseEnvironments}}\n {{.}}{{end}}", "Template string used when commenting on pull requests on Github. The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview].")
-	repoFilter := pflag.StringSlice("filter-repository", []string{}, "Slice with names of repositories which the bot should not respond to.")
+	repoFilter := pflag.StringSlice("blacklist-repositories", []string{}, "Slice with names of repositories which the bot should not respond to.")
```

Comment:
The blacklist term is pretty loaded. Could we maybe use `ignored-repositories` instead?

Reply:
damit, I thought it was great, but ok 🙂